### PR TITLE
vz-5074: Update Weblogic wget to latest download

### PR DESCRIPTION
**Pull request by AbeHern**
Changed the weblogic-deploy-tooling URL to always install the most update-to-date version.

### CODE REVIEW

File: bobs-books/bobs-bookstore-order-manager/deploy/build.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ if [ -f weblogic-deploy.zip ]; then
     echo 'Using existing weblogic-deploy.zip...'
 else
     echo 'Downloading weblogic-deploy.zip...'
-    wget https://github.com/oracle/weblogic-deploy-tooling/releases/download/weblogic-deploy-tooling-1.9.0/weblogic-deploy.zip
+    wget  https://github.com/oracle/weblogic-deploy-tooling/releases/latest/download/weblogic-deploy.zip
```
Comment (anders-swanson) [inline bobs-books/bobs-bookstore-order-manager/deploy/build.sh:37]: nit: can you remove the extra space between wget and the URI

Comment (pshirodkar-orcl) [inline bobs-books/bobs-bookstore-order-manager/deploy/build.sh:37]: Please update the copyright year.

File: bobs-books/bobs-bookstore-order-manager/deploy/build.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+# Copyright (c) 2020, 2021, 2022, Oracle and/or its affiliates.
```
Comment (pshirodkar-orcl) [inline bobs-books/bobs-bookstore-order-manager/deploy/build.sh:2]: We only keep two copyright years maximum - starting year when the file was file was created & latest year when the file was file was updated. Example below:
`# Copyright (c) 2020, 2022, Oracle and/or its affiliates.`

File: bobs-books/bobs-bookstore-order-manager/deploy/build.sh [hunk 3]
```diff
@@ -34,7 +34,8 @@ if [ -f weblogic-deploy.zip ]; then
     echo 'Using existing weblogic-deploy.zip...'
 else
     echo 'Downloading weblogic-deploy.zip...'
-    wget https://github.com/oracle/weblogic-deploy-tooling/releases/download/weblogic-deploy-tooling-1.9.0/weblogic-deploy.zip
+    wget https://github.com/oracle/weblogic-deploy-tooling/releases/latest/download/weblogic-deploy.zip
```
Comment (pfmackin) [inline bobs-books/bobs-bookstore-order-manager/deploy/build.sh:37]: We normally always want to use a specific version when using an artifact.  Double check with Kevin on this.

**Pull request merged**